app.js

```diff
@@ -18,3 +18,130 @@ menuLink.forEach((link) => {
+  languageList: ['html', 'css', 'javascript', 'github', 'ruby', 'bootstrap'],
+  button1Text: 'see live',
+  button2Text: 'see source',
+},
+
+];
+
+const cardContainer = document.querySelector('.cardContainer');
+popUpData.forEach((project) => {
+  const section = document.createElement('section');
+  section.className = 'tonic2';
+  section.innerHTML = `
+          <img src="images/tonic.png" alt="tonic1" class="mob">
+          <img src="imagesdesktop/tonic2-1.png" alt="tonic1" class="desk">
+          <div class="card1">
+            <h2 class="tonic3">${project.Heading}</h2>
+            <ul class="ul2">
+              <li class="can">CANOPY</li>
+              <li class="li2">Back End Dev</li>
+              <li class="li2">2015</li>
+            </ul>
+            <p>A daily selection of privately
+              personalized reads; no accounts or sign-ups
+              required.</p>
+            <ul class="ul3">
+              <li>html</li>
+              <li>css</li>
+              <li>javascript</li>
+              <li class="deskRuby">Ruby on rails</li>
+            </ul>
+
+            <a href="#" class="see myButton">See Project</a>
+          </div>
+       
+         `;
+
+  cardContainer.appendChild(section);
+});
+
+function popUp(project) {
+  const body = document.querySelector('body');
+  const section = document.createElement('section');
+  section.className = 'popup-container';
+  section.innerHTML = ` 
+        <section class="popup-window">
+        <div class="popup-header">
+            <h2 class="tonic3 tonic-head">Tonic  <span class="material-icons cancelPop"> close </span></h2>
+            <ul class="pop-list1">
+              <li class="can">CANOPY</li>
+              <li class="li2">Back End Dev</li>
+            <li class="li2">2015</li>
+           </ul>
+        </div>
+        <div class="popup-img">
+            <img src=${project.imageUrl} alt="project1">
+        </div>
+        <div class="popup-body">
+            <p>${project.paraText}</p>
+            <div class="pop-buttons">
+                <ul class="progrm-l">
+                ${project.languageList.map((link) => ` <li class="pop-lang">${link}</li>`).join('')}               
+                </ul>
+                <div class="pop-hr"></div>
+                <div class="see-popup">
+                    <a href="#"><span>${project.button1Text}</span><img src="Icon - Export.svg" alt="close3"></a>
+                     <a href="#"><span>${project.button2Text}</span><img src="Icon - Export.svg" alt="close3"></a>
```

Comment:
- [ ] This two button should have some margin in between in the desktop version, for now your button has no gap. Please follow the design.

### Your design


![Screenshot from 2021-07-04 00-48-14](https://user-images.githubusercontent.com/22434660/124365048-22e15d80-dc63-11eb-80a3-e75ea647d9c2.png)


### Figma design

![Screenshot from 2021-07-04 00-48-25](https://user-images.githubusercontent.com/22434660/124365045-1e1ca980-dc63-11eb-9ae6-8a709a779d1b.png)

Reply:
:heavy_check_mark: Well Done